lib/member_page.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+require 'scraped'
+require 'date'
+require_relative './member_email_decorator'
+
+class MemberPage < Scraped::HTML
+  decorator Scraped::Response::Decorator::AbsoluteUrls
+  decorator MemberEmailDecorator
+
+  field :id do
+    url.split('/').last
+  end
+
+  field :name do
+    noko.css('.page-header h2').text.tidy
+  end
+
+  field :image do
+    noko.at_css('.item-page img/@src').text
+  end
+
+  field :constituency do
+    table_field('Constituency').match("People's Representative for (.*)")[1]
+  end
+
+  field :birth_date do
+    Date.parse(table_field('Date of Birth')) rescue nil
+  end
+
+  field :gender do
+    table_field('Sex').downcase
+  end
+
+  field :email do
+    noko.xpath("//span[starts-with(@id, 'cloak')]").map(&:text).join(';')
```

Comment:
If you're already decorating the email fields anyway, might be it worth adding some other identifier (e.g. a CSS class) to them so that they're easier to target here?